formats/json/commonTest/src/kotlinx/serialization/features/inline/InlineClassesCompleteTest.kt

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+@file:Suppress("INLINE_CLASSES_NOT_SUPPORTED", "SERIALIZER_NOT_FOUND")
+
+package kotlinx.serialization.features.inline
+
+import kotlinx.serialization.*
+import kotlinx.serialization.Box
+import kotlinx.serialization.test.*
+import kotlin.test.*
+
+@Serializable
+inline class MyInt(val i: Int)
```

Comment:
Can these tests be reused later for other formats?

Reply:
In theory yes, in practice — currently there's no way to move test data to core test source, because json-common-test source set does not depend on core-common-test and can't see declaration in there